docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Query parameters
+
+`owner`::
+(Optional, string or array of strings) A filter to limit the retrieved cases to a
+specific set of applications. Valid values are: `cases`, `observability`,
+and `securitySolution`. If this parameter is omitted, the response contains all
+cases that the user has access to read.
+
+`page`::
+(Optional, integer) The page number to return.
+
+`perPage`::
+(Optional, integer) The number of rules to return per page.
+
+`reporters`::
+(Optional, string) Filters the returned cases by the reporter's `username`.
+
+`search`::
```

Comment:
The original version of these docs in https://www.elastic.co/guide/en/security/master/cases-api-find-cases.html contained the following note:

> Cases are saved objects. See [Find objects API](https://www.elastic.co/guide/en/kibana/master/saved-objects-api-find.html) for more query parameters.

However, the only other query parameter that worked for me was "search" so I've added it here. Are there others that are supposed to work?

Reply:
We support: `defaultSearchOperator`, `fields`, `search`, and `searchFields`. You can see them here:  https://github.com/elastic/kibana/blob/0e7be441400979bf170cf8f74d77c12433da19f3/x-pack/plugins/cases/common/api/cases/case.ts#L136